dll/opengl/glu32/src/libnurbs/internals/subdivider.cc

```diff
@@ -849,8 +849,10 @@ Subdivider::freejarcs( Bin& bin )
 
     Arc_ptr jarc;
     while( (jarc = bin.removearc()) != NULL ) {
-	if( jarc->pwlArc ) jarc->pwlArc->deleteMe( pwlarcpool ); jarc->pwlArc = 0;
-	if( jarc->bezierArc) jarc->bezierArc->deleteMe( bezierarcpool ); jarc->bezierArc = 0;
+	if( jarc->pwlArc ) jarc->pwlArc->deleteMe( pwlarcpool );
+    jarc->pwlArc = 0;
+	if( jarc->bezierArc) jarc->bezierArc->deleteMe( bezierarcpool );
+    jarc->bezierArc = 0;
```

Comment:
Are there real changes here? Or is it just coding style changes?
This is hard to review.
If that's just coding style, I suggest to revert.
Especially since it may be third party synced code.